lib/lambda_ethereum_consensus/beacon/beacon_chain.ex

```diff
@@ -244,9 +244,11 @@ defmodule LambdaEthereumConsensus.Beacon.BeaconChain do
   defp notify_subscribers(logical_time) do
     log_new_slot(logical_time)
 
-    Enum.each([Validator, Gossip.BeaconBlock], fn subscriber ->
+    Enum.each([Gossip.BeaconBlock], fn subscriber ->
       GenServer.cast(subscriber, {:on_tick, logical_time})
     end)
+
+    ValidatorManager.notify_tick(logical_time)
```

Comment:
breaking the pattern here... ValidatorManager is not a genserver so it can't receive cast messages